libs/platform/src/lib/components/form/checkbox/checkbox.component.html

```diff
@@ -0,0 +1,18 @@
+<div fd-form-item>
+    <fd-checkbox
+        [inputId]="id"
+        [name]="name"
+        [disabled]="disabled"
+        [label]="label"
```

Comment:
We need to consult this with somebody, if adding aria tags to this level will really work, as fd-checkbox has yet another layer under this

Reply:
yes these aria tag won't be transmitted to native "input type=checkbox". because core-checkbox is missing this handling. i can check with fundamentals-team if aria-tag can be included. for now if you want i can remove.

Reply:
I think this is fine @mikerodonnell89 what do you think?

Reply:
@stefanoScalzo @mikerodonnell89 , any input on this?

Reply:
We should have some input with arias on `fundamental-ngx`